bentoml/_internal/bento/build_config.py

```diff
@@ -483,7 +483,7 @@ def write_to_bento(self, bento_fs: FS, build_ctx: str) -> None:
             install_script_content = (
                 """\
 #!/usr/bin/env bash
-set -ex
+set -exuo pipefail
```

Comment:
@aarnphm let's wait for another review if you made changes to the PR

Reply:
sg